scripts/tools/cancel_workflows_for_pr.py

```diff
@@ -32,28 +37,95 @@
 
 
 class Canceller:
-    def __init__(self, token):
+    def __init__(self, token: str, dry_run: bool):
         self.api = Github(token)
         self.repo = self.api.get_repo(REPOSITORY)
+        self.dry_run = dry_run
+
+    def check_all_pending_prs(self, max_pr_age_days, required_runs):
+        for pr in self.repo.get_pulls(state="open", sort="updated", direction="desc"):
+            pr_create = (
```

Comment:
super nit, `pr_create` -> `pr_update_time`

Reply:
Updated in a followup commit.